### ISSUE TITLE
Can't set focus inside debug complex hover tooltip 

### ISSUE THREAD

**Issue by stevencl** [Microsoft/vscode#3293]
With accessibility mode on and tab set to move focus, I can't seem to use the keyboard to navigate to or inside the tooltip that appears when hovering over a complex object in the debugger.

Found while testing #3246

**Comment by isidorn** [comment]
@stevencl did you try ctrl + k, ctrl + i. Can you move focus using the same keyboard shortuct to the regular, non debug hover? Can you move focus inside a simple debug hover?
This works fine for me on the mac

**Comment by stevencl** [comment]
I didn't use Ctrl K, Ctrl I originally, wasn't aware of that. After clicking inside the tooltip with the mouse then pressing Ctrl K, Ctrl I, I am able to move focus inside the tooltip.
But, I wonder how useful this will be? By requiring the user to use the mouse to reveal the hover and then click inside it, it seems like we are not fully enabling keyboard accessibility.
Perhaps though there is another keyboard shortcut that I am not aware of that brings up the tooltip for the symbol located next to the cursor?

**Comment by isidorn** [comment]
I think there is a misunderstanding, <kbd> ctrl k cltr i</kbd> is the keyboard shortcut that brings up the tooltip for the symbol located next to the cursor. You do not need to use the mouse at all!

This keybinding also works for regular hover - same way as visual studio is doing it.

If this does not work for you please reopen the bug.

**Comment by stevencl** [comment]
D'oh! I don't know why I didn't see this before.

**Comment by isidorn** [comment]
No problemo, just glad we got the confusion out of the way 🐙
